Client/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.16.0 FATAL_ERROR)
 project(client)
 
-add_executable(client main.cpp Client.cpp Renderer.cpp TutorialGame.cpp Font.cpp)
+add_executable(client main.cpp Client.cpp Renderer.cpp TutorialGame.cpp Font.cpp DebugMode.cpp)
```

Comment:
There's a better way to do this, will send a corresponding message/show in lab. This is a fine way though